pytorch_lightning/callbacks/base.py

```diff
@@ -264,7 +264,12 @@ def on_predict_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule")
         pass
 
     def on_keyboard_interrupt(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -> None:
```

Comment:
Thinking we shall have better check if a user has a custom callback using deprecated hook...
So it can be something like inspection that callback has a method which was in the past valid hook?
cc: @PyTorchLightning/core-contributors

Reply:
@Borda there's a similar suggestion discussed here: https://github.com/PyTorchLightning/pytorch-lightning/discussions/9070